templates/katello.yaml.erb

```diff
@@ -24,7 +21,6 @@
     :ssl_ca_file: <%= @candlepin_ca_cert %>
 
   :pulp:
-    :url: <%= @pulp_url %>
     :ca_cert_file: <%= @pulp_ca_cert %>
 
   :use_pulp_2_for_content_type:
```

Comment:
Feels like this could be removed as well

Reply:
:+1: